examples/apache-directory-list-local/webdav.conf

```diff
@@ -1,23 +1,16 @@
-ServerName webdav.server.com
```

Comment:
Is it worth keeping this in to squelch the error message in the Apache startup?

Reply:
Sure I wasn't sure if it would work

Reply:
https://github.com/dom111/webdav-js/pull/21/commits/3bff5b831d6ba5a19618416ed3bfa6de8634bd8c done

Reply:
Lol it doesn't work: with `webdav.server.com` gives the same error

This is not fixable.

Reply:
Ah, I used `ServerName localhost`, that worked!

Reply:
Strange I'm getting this no matter what ServerName I add :confused: :
```
apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
```
https://stackoverflow.com/questions/46266527/could-not-reliably-determine-the-servers-fully-qualified-domain-name-how-to

Looks like it has to do with the hosts file. I'm sticking with `webdav.server.com` you can change it after but it honestly doesn't matter in my opinion.

Reply:
Ok cool, I might have a play with this post-merge, thanks for investigating!